packages/devextreme/js/__internal/ui/chat/messagelist.ts

```diff
@@ -71,7 +81,12 @@ class MessageList extends Widget<Properties> {
 
     this._renderScrollView();
     this._renderMessageListContent();
+    this._renderEmptyView();
```

Comment:
Did you consider scenario when Chat is empty and some users are typing simultaneously, what layout should be in this case?

Reply:
Yes reviewed, there are TestCafe tests on these scenarios and new screenshots added.